plugins/ccgx/ccgx.quirk

```diff
@@ -51,13 +51,25 @@ Summary = CCGx Power Delivery Device (Symmetric FW1)
 [USB\VID_04B4&PID_5218&SID_1F00&APP_6432&MODE_FW2]
 Summary = CCGx Power Delivery Device (Symmetric FW2)
 
+# Lenovo ThinkPad USB-C Dock Viking
+[USB\VID_17EF&PID_30A9]
+Plugin = ccgx
+GType = FuCcgxDmcDevice
+Summary = Dock Management Controller Device
+ParentGuid = USB\VID_17EF&PID_30AF
+Name = Lenovo ThinkPad USB-C Dock Viking
```

Comment:
I guess based on Mark's comments this should just be "USB-C Dock".  If later this is made POR it can be renamed to include some branding.

Reply:
> I guess based on Mark's comments this should just be "USB-C Dock". If later this is made POR it can be renamed to include some branding.

I will change Name as USB-C Dock.

@mrhpearson 
Wistron asked me to develop LVFS for Lenovo Viking dock three weeks ago.
So I requested code review as a prototype. There will be many things to be defined.
Let me know your email in Lenovo so that l make email for this issue.

Reply:
Please check [53c653c](https://github.com/fwupd/fwupd/pull/4435/commits/53c653ca9db90a142b95992e167f06f819396a0f)